### PR TITLE
Added the possibility to use waas or universal wallet

### PR DESCRIPTION
**Pull request by RichardIrala**
I need someone to test if it works well for them. And I need them to improve it if necessary.

### CODE REVIEW

**Comment by RichardIrala** [comment]
* pnpm lint (doesn't work)
* pnpm build (doesn't work)

File: src/env.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ export const env = createEnv({
     NEXT_PUBLIC_SEQUENCE_ACCESS_KEY: z.string(),
     NEXT_PUBLIC_SEQUENCE_PROJECT_ID: z.string(),
     NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID: z.string().optional(),
+    NEXT_PUBLIC_WAAS_CONFIG_KEY: z.string().optional(),
+    NEXT_PUBLIC_GOOGLE_CLIENT_ID: z.string().optional(),
+    NEXT_PUBLIC_WALLET_TYPE: z.enum(["waas", "universal"]),
```
Comment (AlexanderKolberg) [inline src/env.js]: Can we make this optional and add a default here?

File: src/config/networks/wagmi/index.ts [hunk 2]
```diff
@@ -16,32 +16,54 @@ import {
...
 import { findNetworkConfig, allNetworks } from '@0xsequence/network';
 import type { Chain, Transport } from 'viem';
 import { createConfig, http } from 'wagmi';
 
-const accessKey = env.NEXT_PUBLIC_SEQUENCE_ACCESS_KEY;
+const accessKey = env.NEXT_PUBLIC_SEQUENCE_ACCESS_KEY!;
+const walletConnectProjectId = env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID!;
+const waasConfigKey = env.NEXT_PUBLIC_WAAS_CONFIG_KEY!;
+const googleClientId = env.NEXT_PUBLIC_GOOGLE_CLIENT_ID!;
+const walletType = env.NEXT_PUBLIC_WALLET_TYPE!;
```
Comment (AlexanderKolberg) [inline src/config/networks/wagmi/index.ts]: Lets not use non-null assertion here, it brakes the typesafety

File: src/env.js (hunk 1 above)
Comment (RichardIrala) [reply src/env.js]: Yes, modify it if you want

**Pull request merged**